### DISCUSSION TITLE
About RoIPool in mmcv

### DISCUSSION THREAD

**Discussion by ghost** [open-mmlab/mmcv#2429] [General]
Thank you for the great work you provide！

I would like to know if maxpool or avgpool is used in the RoIPool module？ I found that RoIAlign is providing the interface for us to choose, but it seems to be missing in RoIPool as well as RoIAlignRotated

Can I use the MaxPool operation in the RoIAlignRotated module?

Looking forward to your reply!

**Comment by zhouzaida** [comment]
Hi, `maxpool` is used in `RoIPool` module.
